package.json

```diff
@@ -42,7 +42,7 @@
     "debug": "~3.1.0",
     "human-interval": "~0.1.3",
     "moment-timezone": "~0.5.0",
-    "mongodb": "~2.2.33"
+    "mongodb": "~3.0.10"
```

Comment:
Let's bump to 3.1 since we need that for MongoDB 4 support

Reply:
No objections but I also think if it causes any extra work and slows down merging, it's smart to keep leave that for a separate PR.